EIPS/eip-1.md

```diff
@@ -1,101 +1,100 @@
+  * :x: -- A Last Call which results in material changes or substantial unaddressed complaints will cause the EEP to revert to Draft.
+  * :arrow_right: Accepted (Core EEPs only) -- After the review end date, the [IF WE HAVE SOMETHING SIMILAR TO CORE DEVS] EOS Core Developers will vote on whether to accept this change. If yes, the status will upgrade to Accepted.
+  * :arrow_right: Final (Not core EEPs) -- A successful Last Call without material changes or unaddressed complaints will become Final.
+* **Accepted (Core EEPs only)** -- This is being implemented by [FLAGGING FOR CORE DEV] EOS Core Developers.
+  * :arrow_right: Final -- Standards Track Core EEPs must be implemented in at least three viable EOS clients [WILL WE NEED A DIFFERENT AMOUNT?] before it can be considered Final. When the implementation is complete and supported by the community, the status will be changed to “Final”.
+* **Final** -- This EEP represents the current state-of-the-art. A Final EEP should only be updated to correct errata.
 
 Other exceptional statuses include:
 
-* Deferred -- This is for core EIPs that have been put off for a future hard fork.
-* Rejected -- An EIP that is fundamentally broken and will not be implemented.
-* Active -- This is similar to Final, but denotes an EIP which which may be updated without changing its EIP number.
-* Superseded -- An EIP which was previously final but is no longer considered state-of-the-art. Another EIP will be in Final status and reference the Superseded EIP.
+* Deferred -- This is for core EEPs that have been put off for a future hard fork.
+* Rejected -- An EEP that is fundamentally broken and will not be implemented.
+* Active -- This is similar to Final, but denotes an EEP which which may be updated without changing its EEP number.
+* Superseded -- An EEP which was previously final but is no longer considered state-of-the-art. Another EEP will be in Final status and reference the Superseded EEP.
 
-## What belongs in a successful EIP?
+## What belongs in a successful EEP?
 
-Each EIP should have the following parts:
+Each EEP should have the following parts:
 
-- Preamble - RFC 822 style headers containing metadata about the EIP, including the EIP number, a short descriptive title (limited to a maximum of 44 characters), and the author details. See [below](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-header-preamble) for details.
-- Simple Summary - “If you can’t explain it simply, you don’t understand it well enough.” Provide a simplified and layman-accessible explanation of the EIP.
+- Preamble - RFC 822 style headers containing metadata about the EEP, including the EEP number, a short descriptive title (limited to a maximum of 44 characters), and the author details. See [LEAVING THIS IN AS WE'LL NEED TO CREATE OUR OWN VERSION AND THEN EDIT THIS LINK] [below](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-header-preamble) for details.
```

Comment:
Not sure who will hold the final EEP repository, but I would put `https://github.com/eoscanada/EEPs/blob/master/EEPS/eep-1.md`

Also, you will need to update `eips/eip-1.md` to `eeps/eep-1.md` once we merged this PR.